css/style.css

```diff
@@ -0,0 +1,55 @@
+#navbar {    
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 175px;
+    min-width: 150px;
+    height: 100%;
+    padding: 40px 0 0 0;
+    border: 2px solid;
+    text-align: center;
+}
+
+#logo {
+    width: 80px;
+    height: 80px;
+    display: relative;
+    margin:;    
+}
+
+#main-doc {
+    margin-left: 200px;
+}
```

Comment:
It would be better if you use classes instead of id's since id's limit you to use them in only one element.